packages/i18n/lib/index.mjs

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-import localeConfig from '@node-core/website-i18n/config.json' assert { type: 'json' };
+import localeConfig from '@node-core/website-i18n/config.json' with { type: 'json' };
```

Comment:
Any chance that this behaves differently on staging and somehow fails to import the locales correctly?

Reply:
We use 20 on vercel and we have set nvmrc to 20.

Reply:
Thanks! Interesting, I'm using v20.11.1 locally, worked fine for me. Maybe this isn't the issue we're seeing on staging, but maybe some logging as mentioned in https://github.com/nodejs/nodejs.org/pull/7155#issuecomment-2451394727 above could help to diagnose the error.